.github/workflows/image-prs.yaml

```
@@ -0,0 +1,118 @@
+name: Image PRs Build
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-push-prs:
+    if: ${{ github.repository == 'cilium/ubuntu-curl' }}
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - name: ubuntu-curl
+            dockerfile: ./Dockerfile
+
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@94ab11c41e45d028884a99163086648e898eed25
+
+      - name: Login to quay.io for CI
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_CI_ALPINE_CURL_USERNAME }}
```

can we reuse the same credentials here or should we create a new one

---

Let's swap to 
```suggestion
          username: ${{ secrets.QUAY_CI_UBUNTU_CURL_USERNAME }}
```

(and swap ALPINE -> UBUNTU everywhere else too)